accounts/models.py

```diff
@@ -1,2 +1,30 @@
-# from django.db import models
-# from django.contrib.auth.models import User
+from django.db import models
+from django.contrib.auth.models import User
+# Each user can add, modify or remove course from is list
+# Each course contains coures id(default by Django), name, credinitials and grade
+
+
+class Course (models.Model):
+    user = models.ManyToManyField(User, blank=True)
```

Comment:
Is that really the relation?
Will a record of Course (is it the right name?) that includes a field like "grade" is going to be associated with several users?

Reply:
No ,the main idea was that each user can have many courses an and each course can be in many users.
this is way i put many to many.

Reply:
The current implementation makes it so every student will have the same instance of the course, including the same grade.
A More suitable implementation would be a table for individual courses and table/s describing the relation of those courses to the students  (maybe something along the lines of - `grades`, or `student-in-course`, etc)

Reply:
@Omeramsc  so if I understand what you saying , you suggest that I will do an additional membership class exemple " student in coure" and then to make ForeignKey(courese) and ForeignKey(user)?